### ISSUE TITLE
This again... Uncaught TypeError Undefined is not a function and another one...

### ISSUE THREAD

**Issue by alchzh** [sedabull/quantum-shell#17]
I am fully aware this is like the third time this has been brought up (#14 #13)
However, I am still getting this error.
Oh, and it seems that the only command I can use it `pwd`. Everything gives me a /bin/sh enoent error...

(full traceback)

```
TypeError: undefined is not a function
  at QuantumShellModel.module.exports.~cd (C:\Users\Albert\.atom\packages\quantum-shell\lib\builtins\sh.coffee:53:47)
  at QuantumShellModel.process (C:\Users\Albert\.atom\packages\quantum-shell\lib\quantum-shell-model.coffee:245:41)
  at HTMLButtonElement.<anonymous> (C:\Users\Albert\.atom\packages\quantum-shell\lib\quantum-shell-controller.coffee:94:26)
  at HTMLInputElement.subscriptions.add.atom.commands.add.quantum-shell:submit (C:\Users\Albert\.atom\packages\quantum-shell\lib\quantum-shell-controller.coffee:128:60)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (c:\Users\Albert\AppData\Local\atom\app-0.211.0\resources\app.asar\src\command-registry.js:241:29)
  at c:\Users\Albert\AppData\Local\atom\app-0.211.0\resources\app.asar\src\command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (c:\Users\Albert\AppData\Local\atom\app-0.211.0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (c:\Users\Albert\AppData\Local\atom\app-0.211.0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (c:\Users\Albert\AppData\Local\atom\app-0.211.0\resources\app.asar\src\window-event-handler.js:180:20)
```

O wait! I think I found something. Looking at the developer console I see this...
`ENOENT: no such file or directory, scandir 'C:\Panda3D-1.9.0-x64\bin:\usr\local\bin'`

WAIT WHAT! When did \usr\local\bin get into my PATH? (/usr/local/bin is not normally in my path)

(If you couldn't tell by the traceback, I'm on Windows 8.1)

**Comment by alchzh** [comment]
Ok, deleted some deprecated packages, but I still can't run `cd`

**Comment by hoosierhobbyist** [comment]
This is related to a refactoring of the `cd` builtin in the last version, which now relies on UNIX like file permissions to check if a user has the proper clearance to enter a directory (It's much more robust this way). Unfortunately, this approach doesn't work on windows like the old one did and I didn't realize I had so many windows users. I will try to fix this soon, however it may be over a week before I have time.

Also, this is more related to #16 than #13 and #14. `uncaught TypeError` is pretty broad, and I've found that Atom isn't very good at detecting if an error has _really_ already been reported. Sometimes while I'm developing, I create a new error that could not _possibly_ have already been reported, and it still asks me if I want to view the issue on github. So whenever you run into a problem don't hesitate to open a new issue and I'll get to it as soon as I can.

**Comment by alchzh** [comment]
Ah, that's fine. I'm not usually on Win, but it is annoying when I am :/

**Comment by hoosierhobbyist** [comment]
This issue should be resolved as of v-0.7.0, please let me know if any problems persist or there are any other issues!
